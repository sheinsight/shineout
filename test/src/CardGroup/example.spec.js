import exampleTest from '../../example'

exampleTest('CardGroup')
