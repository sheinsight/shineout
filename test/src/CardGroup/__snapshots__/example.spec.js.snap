// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: CardGroup[site/pages/components/CardGroup/example-1-base.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Tags
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Owner
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Property
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Score
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Deal
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Task
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Send
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Info
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Success
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Filter
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshot test: CardGroup[site/pages/components/CardGroup/example-2-width.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Tags
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Owner
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Property
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Score
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Deal
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Task
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Send
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Info
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Success
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Filter
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshot test: CardGroup[site/pages/components/CardGroup/example-3-lazyload.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Tags
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Owner
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Set Property
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Score
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Deal
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Add Task
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Send
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Info
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Success
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
      <div>
        <div>
          <div>
            <div>
              <i />
            </div>
            <span>
              Filter
            </span>
          </div>
          <p>
            Add or delete tag for your customer. You can sort your customer...
          </p>
          <div>
            <i />
            Add This
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshot test: CardGroup[site/pages/components/CardGroup/example-4-checkbox.tsx] 1`] = `
<div>
  <p>
    <label>
      <input />
      <i />
      <span>
        全选
      </span>
    </label>
  </p>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Tags
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Set Owner
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Set Property
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Score
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Add Deal
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Add Task
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Send
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Info
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Success
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Filter
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshot test: CardGroup[site/pages/components/CardGroup/example-5-disabled.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Tags
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Set Owner
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Set Property
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Score
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Add Deal
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Add Task
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Send
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Info
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Success
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
        <div>
          <div>
            <div>
              <div>
                <i />
              </div>
              <span>
                Filter
              </span>
            </div>
            <p>
              Add or delete tag for your customer. You can sort your customer...
            </p>
            <div>
              <i />
              Add This
            </div>
          </div>
          <label>
            <input />
            <i />
          </label>
        </div>
      </div>
    </div>
  </div>
</div>
`;
